state/human.context.js

```diff
@@ -57,6 +69,7 @@ function HumanProvider(props) {
   }
 
   const confirmProposal = async ({ proposalId, code }) => {
+    if (!humanSDK.isReady()) return
```

Comment:
this guard is [already in the SDK](https://github.com/Tutellus/humanWalletSDK/blob/develop/src/index.js#L160), so you don't have to care about here